cloudinit/url_helper.py

```diff
@@ -193,7 +193,7 @@ def _get_ssl_args(url, ssl_details):
 def readurl(
     url,
     data=None,
-    timeout=None,
+    timeout=30,
```

Comment:
I found today that the [EC2 datasource](https://github.com/canonical/cloud-init/blob/main/cloudinit/sources/DataSourceEc2.py#L90) sets a default timeout to 50, so I think we should bump this up to 60 as a global default.

Reply:
50s? Are their server rooms full of carrier pigeons??? 
Just kidding. I agree 60s is good!